src/app/constants/telemetry.ts

```diff
@@ -27,7 +27,22 @@ export enum TELEMETRY_PAGE_NAMES {
 
 export enum TELEMETRY_EVENTS {
     FETCH_DEVICES = 'fetch_devices',
-    INIT = 'init',
+    INTERNAL_USER = 'internal_user',
     UPDATE_BANNER_DISPLAYED = 'update_banner:displayed',
-    UPDATE_BANNER_CLICKED = 'update_banner:clicked'
+    UPDATE_BANNER_CLICKED = 'update_banner:clicked',
+    USER_ACTION = 'user_action'
+}
+
+export enum TELEMETRY_USER_ACTIONS {
+    UPDATE_DEVICE_TWIN = 'update_device_twin',
+    START_DEVICE_TELEMETRY = 'start_device_telemetry',
+    DEVICE_INVOKE_DIRECT_METHOD = 'device_invoke_direct_method',
+    SEND_C2D_MESSAGE = 'send_c2d_message',
+    ADD_DEVICE = 'add_device',
+    ADD_MODULE = 'add_module',
+    UPDATE_MODULE_TWIN = 'update_module_twin',
+    MODULE_INVOKE_DIRECT_METHOD = 'module_invoke_direct_method',
+    PNP_SEND_COMMAND = 'pnp_send_command',
+    PNP_UPDATE_PROPERTY = 'pnp_update_property',
+    START_PNP_TELEMETRY = 'start_pnp_telemetry'
```

Comment:
let's rename START_PNP_TELEMETRY to PNP_ START_TELEMETRY so all pnp actions are the same prefix, easier to view later.